<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisAlignedBoxParametersControlWidget</class> 
 <widget class="QWidget" name="AxisAlignedBoxParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AxisAlignedBoxParameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>X</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="xSpinBox">
       <property name="toolTip">
         <string>X</string>
       </property>
       <property name="minimum">
        <double>-50000</double>
       </property>
       <property name="maximum">
        <double>50000</double>
       </property>
       <property name="singleStep">
        <double>100</double>
       </property>
       <property name="value">
        <double>0</double>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>Y</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="ySpinBox">
       <property name="toolTip">
         <string>Y</string>
       </property>
       <property name="minimum">
        <double>-50000</double>
       </property>
       <property name="maximum">
        <double>50000</double>
       </property>
       <property name="singleStep">
        <double>100</double>
       </property>
       <property name="value">
        <double>0</double>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>Z</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="zSpinBox">
       <property name="toolTip">
         <string>Z</string>
       </property>
       <property name="minimum">
        <double>-50000</double>
       </property>
       <property name="maximum">
        <double>50000</double>
       </property>
       <property name="singleStep">
        <double>100</double>
       </property>
       <property name="value">
        <double>3000</double>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>width</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="widthSpinBox">
       <property name="toolTip">
         <string>width</string>
       </property>
       <property name="minimum">
        <double>-50000</double>
       </property>
       <property name="maximum">
        <double>50000</double>
       </property>
       <property name="singleStep">
        <double>100</double>
       </property>
       <property name="value">
        <double>1000</double>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
       <property name="text">
         <string>height</string>
       </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="heightSpinBox">
       <property name="toolTip">
         <string>height</string>
       </property>
       <property name="minimum">
        <double>-50000</double>
       </property>
       <property name="maximum">
        <double>50000</double>
       </property>
       <property name="singleStep">
        <double>100</double>
       </property>
       <property name="value">
        <double>1000</double>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
       <property name="text">
         <string>depth</string>
       </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="depthSpinBox">
       <property name="toolTip">
         <string>depth</string>
       </property>
       <property name="minimum">
        <double>-50000</double>
       </property>
       <property name="maximum">
        <double>50000</double>
       </property>
       <property name="singleStep">
        <double>100</double>
       </property>
       <property name="value">
        <double>1000</double>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>xSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>AxisAlignedBoxParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>ySpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>AxisAlignedBoxParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>zSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>AxisAlignedBoxParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>widthSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>AxisAlignedBoxParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>heightSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>AxisAlignedBoxParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>depthSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>AxisAlignedBoxParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
